### DISCUSSION TITLE
support for auto-numbering of lists

### DISCUSSION THREAD

**Discussion by metronome12** [gonzalezreal/swift-markdown-ui#192] [Q&A] [answered]
In markdown I can create a number list like this: 

`1`. item 1
`1`. item 2
`1`. item 3

and it will auto number them like this

1. item 1
2. item 2
3. item 3

It would be good to provide that in MarkdownUI

**Comment by gonzalezreal** [comment, marked as answer]
MarkdownUI 2.x is compatible with the GitHub Flavored Markdown spec, which includes the CommonMark spec and, thus, the auto-numbering lists feature. You can see this in the [companion demo project](https://github.com/gonzalezreal/swift-markdown-ui/blob/main/Examples/Demo) under the **Lists** screen in the Customization Example section:

https://github.com/gonzalezreal/swift-markdown-ui/blob/fdc444dba7b9d89f3581b480ce919d3d07f38aa3/Examples/Demo/Demo/ListsView.swift#L68-L70

In the example, there is an auto-numbered list that starts at 10 and is styled with roman numbers.

Please, check the demo project and the docs to get acquainted with all the library features.
